build-farm/platform-specific-configurations/linux.sh

```diff
@@ -43,7 +43,7 @@ fi
 if [ "${VARIANT}" == "${BUILD_VARIANT_OPENJ9}" ]
 then
   # CentOS 6 has openssl 1.0.1 so we use a self-installed 1.0.2 from the playbooks
-  if grep 'release 6' /etc/redhat-release >/dev/null; then
+  if grep 'release 6' /etc/redhat-release >/dev/null || grep 'jessie' /etc/os-release >/dev/null; then
```

Comment:
Add 2>&1 after each `>/dev/null` otherwise it'll chuck out errors when theat doesn't exist (I suspect it is throwing such an error on non-RHEL6 systems already)

Reply:
Done 👍